snap/snapcraft.yaml

```diff
@@ -13,21 +13,23 @@ apps:
   srsepc:
```

Comment:
(not this line) the top level `version` needs to be updated as well

Reply:
definitely :)
✅